lametro/models.py

```diff
@@ -825,8 +828,23 @@ def is_ready(self):
         return self.ready
 
     def _merge_docs(self):
-        merge_url = settings.MERGER_BASE_URL + '/merge_pdfs/' + self.related_entity.slug
-        requests.post(merge_url, json=self.related_files)
+        headers = {
+            'cache-control': 'no-cache',
+            'content-type': 'application/json',
```

Comment:
if you use the `json` argument in `request.post` you should not need to specify the content-type

why do we need cache-control here?

Reply:
Not sure. `cache-control` was in the example request in the Airflow docs: https://airflow.apache.org/docs/apache-airflow/stable/deprecated-rest-api-ref.html#endpoints

Reply:
by default, the requests methods don't respect anything related to caching headers, so we can omit that too.

Reply:
Got it, removed the headers entirely in https://github.com/datamade/la-metro-councilmatic/pull/766/commits/13c5726ef17ef7777c46a70d65626738309b3fd3.